src/components/NostoProduct/common.ts

```diff
@@ -0,0 +1,17 @@
+import { Store } from "./store"
+
+export function syncPrices(element: HTMLElement, setter: Store["setPrices"]) {
+  const price = element.getAttribute("n-price")
```

Comment:
Consider adding dev mode? 
Also, should we continue receiving all SKU/product data through attributes? When we extend the web-components to support more complex cases requiring more data, receiving it through attributes may become unmanageable.

Reply:
What do you mean by dev mode? Adjust the dev example to have prices?

Reply:
I agree regarding the data scalability. Should we add SKU level data support next?

Reply:
Yes, SKU level data support should be improved

Reply:
Yes, prices in dev mode would help us test it locally